common/constants.js

```diff
@@ -1,4 +1,5 @@
-export const BASE_URL = 'https://api.doordeck.com'
+export const DEFAULT_BASE_URL = 'https://api.doordeck.com'
+export const BASE_URL = '4zA64daKfi'
 export const EPHEMERAL_PRIKEY = 'S8Y0bEpu4y'
 export const EPHEMERAL_PUBKEY = '4yA64dfKfi'
 export const AUTH_TOKEN = '2uGGg077YR'
```

Comment:
What are these?

Reply:
These are created since 2019 and at the moment they're used as keys for the local storage (map).
I believe there's no immediate harm (users would have to re-log in I think) if these are changed

![Uploading image.png…]()

Reply:
This seems to be a key obfuscation for security reasons. Though, it is not a realistic defense but no harm to keep or not keep them.